frontend/javascripts/oxalis/view/action-bar/share_modal_view.js

```diff
@@ -119,7 +119,7 @@ export default function ShareModalView(props: Props) {
         "The dataset of this annotation is not public. The Sharing Link will make the dataset accessible to everyone you share it with.";
     } else if (visibility === "Private") {
       message =
-        "The annotation is currently private, so Team Sharing is disabled and only admins and team managers can use the Sharing Link.";
+        "The annotation is currently private, so Team Sharing is disabled and only admins and dataset managers can use the Sharing Link.";
```

Comment:
Reminder to test that this actually works for dataset managers and doesn't work for team managers.

Reply:
@daniel-wer This change was a mistake. Private shared annotations should only be visible to admins or team managers but not to dataset managers. 

@hotzenklotz You wanted to test this PR, correct? Could you please test this too?